actors/market/tests/cron_tick_deal_expiry.rs

```diff
@@ -0,0 +1,218 @@
+// Copyright 2019-2022 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use fil_actors_runtime::network::EPOCHS_IN_DAY;
+use fil_actors_runtime::runtime::Policy;
+use fvm_shared::clock::ChainEpoch;
+
+use num_traits::Zero;
+
+mod harness;
+use harness::*;
+
+const START_EPOCH: ChainEpoch = 50;
+const END_EPOCH: ChainEpoch = START_EPOCH + 200 * EPOCHS_IN_DAY;
+const SECTOR_EXPIRY: ChainEpoch = END_EPOCH + 400;
+
+#[test]
+fn deal_is_correctly_processed_twice_in_the_same_crontick() {
```

Comment:
This test survived through market actor changes that make all of its editorializing false:

1. name should be: deal_is_correctly_processed_if_first_cron_after_expiry (there is no processing twice, just once)
2. line 31 remove "and scheduled next epoch at end epoch -1", that no longer happens its still scheduled to be processed at 2880 from the publish storage deals call
3. line 41 remove "so deal is first processed at endEpoch - 1 AND then at it's end epoch" (its only ever processed at 2880)